shared/middleware/files_api.rb

```diff
@@ -24,6 +26,10 @@ def get_bucket_impl(endpoint)
 
   def allowed_file_type?(endpoint, extension)
     case endpoint
+    when 'animations'
+      # Only allow specific image types to be uploaded by users.
+      # Only png for now, will expand support in the future
+      %w(.png).include? extension.downcase
```

Comment:
Good call.  The only other `case endpoint` I see in this file is the caching times (line 127), maybe I'll give that the same treatment.